tasks/request_files/test/unit_tests/test_request_files.py

```diff
@@ -639,7 +639,7 @@ def test_object_exists_client_error_raised(self):
             self.assertEqual(expected_error, err)
 
     def test_object_exists_NotFound_returns_false(self):
-        expected_error = ClientError({'Error': {'Code': 'NotFound'}}, '')
+        expected_error = ClientError({'Error': {'Code': '404', 'Message': 'Not Found'}}, '')
```

Comment:
Is Not Found in message supposed to have a space or should there be no space like NotFound?

Reply:
I am seeing "Not Found" in the error message.

{"message": "An error occurred (404) when calling the HeadObject operation: Not Found", "timestamp": "2021-05-24T14:28:07.521580", "level": "error"} {'Error': {'Code': '404', 'Message': 'Not Found'}